addons/event/models/event_event.py

```diff
@@ -203,7 +203,7 @@ def _default_description(self):
     badge_innerright = fields.Html(string='Badge Inner Right')
     event_logo = fields.Html(string='Event Logo')
 
-    @api.depends('stage_id', 'kanban_state')
```

Comment:
Why this change?

Since we use the stage_id to compute the label, I think it can stay there.
Looks like all other implementations of kanban_state_label specify stage_id in the depends as well.